printer_mods/chrisrgonzales/270_degree_hinge/README.md

```diff
@@ -51,7 +51,7 @@ M3 Threaded Insert   [x8]
 ```
 
 ## Printing
-Use the default recommend Voron parts. Can be done in PLA but i have not tested this. If having issues printing side mount
+Use the default recommended Voron parts. Can be done in PLA but I have not tested this. If having issues printing side mounts
```

Comment:
Sorry, I missed this before - change `recommended Voron parts` to `recommended for Voron parts`.

Reply:
All good you have a good idea for catching it.